best-practices/check_deprecated_apis.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: check-deprecated-apis
+  annotations:
+    policies.kyverno.io/category: Best Practices
+    policies.kyverno.io/description: >-
+      Kubernetes APIs are sometimes deprecated and removed after a few releases. 
+      As a best practice older API versions should be replaced with newer versions.
+spec:
+  validationFailureAction: enforce
```

Comment:
I'm trying to standardize that all validate policies are set to `audit`. This is to ensure no unintended blocking happens if users want to take a policy for a "spin" and don't quite realize what may happen.

Reply:
Sounds good. Updated.